aten/src/ATen/miopen/Descriptors.cpp

```diff
@@ -109,9 +109,25 @@ void FilterDescriptor::set(const at::Tensor &t, int64_t pad) {
   for (int i = dim; i < pad; ++i) {
     size[i] = (int) 1;
   }
-  for (int i = dim - 1; i >=0; --i) {
-    stride[i] = (i == dim - 1) ? 1 : stride[i+1] * size[i+1];
+  if( memory_format != at::MemoryFormat::ChannelsLast ) {
```

Comment:
Don't do this. Do a switch on memory format and explicitly error if it is an unexpected error; this will make the code robust if a new memory format gets added.

Reply:
Updated; this is gone.